web/src/app/components/survey-list/survey-list.component.ts

```diff
@@ -64,4 +68,21 @@ export class SurveyListComponent implements OnInit, OnDestroy {
   ngOnDestroy(): void {
     this.subscription.unsubscribe();
   }
+
+  private isSetupFinished(survey: Survey): boolean {
```

Comment:
I would like it to be in a more common place so that I don't have to repeat it twice. But then the router or navigation service would have to get the survey to know where to route/navigate... Let's discuss

Reply:
Add a state to survey, draft, complete, (testing).

if draft, go to creation flow 1st step with draft survey (from list/edit page)
if complete, go to edit page from create page, go to map page from list page

Reply:
Created bug #1171